packages/gantt/src/class/item.ts

```diff
@@ -46,6 +47,7 @@ export class GanttItemInternal {
     links: GanttLink[];
     color?: string;
     barStyle?: Partial<CSSStyleDeclaration>;
+    cellStyle?: Partial<CSSStyleDeclaration>;
```

Comment:
Thanks for the PR! Since each bar occupies a full row in the Gantt chart, I think `laneStyle` is a more accurate name than cellStyle. It better reflects the structure and avoids confusion with table cells.